src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -75,6 +50,57 @@ export default function () {
 
 </CodeGroup>
 
+
+## Module Properties
+
+Listed in the table are the importable properties from the top level module (`'k6/x/browser'`).
+
+| Properties | Description                                                                                                                                                                                                                         |
+|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| chromium   | A [BrowserType](/javascript-api/xk6-browser/browsertype) to launch tests in a chromium browser.                                                                                                                                     |
+| devices    | Returns predefined emulation settings for many end-user devices; see the list of [supported devices](#supported-devices). It can be used to simulate browser behavior on a mobile device. See the [example here](#devices-example). |
+| version    | Returns the version number of xk6-browser.                                                                                                                                                                                          |
+
+### Supported Devices
+
+These are the supported devices for simulating browser behavior on mobile devices. Use the names exactly as listed. To test in landscape mode, add `" landscape"` to the end of the name e.g. `"iPad landscape"`.
```

Comment:
Do you think this is useful to document? `devices` is a map that users can inspect on their own, so maybe just the example should be enough. Besides, these are all quite old devices, which raises the question of when will modern devices be added. And once we start [updating the settings automatically](https://github.com/grafana/xk6-browser/issues/290), it would be a chore to keep the list here in sync.

So I'd vote for removing this section and just keeping the example.

Reply:
I was on the fence with this one too, so happy to remove it.

Resolved in: e9ced7265ed08fed9b7b203df46a3025e172824c